spec/core/kuniri_spec.rb

```diff
@@ -10,6 +10,34 @@
       folderFiles = Dir[File.join("./", "**", "**.rb")]
       expect(defaultPath).to match_array(folderFiles)
     end
+
   end
 
+  context "When source directory is empty" do
+    before :each do
+      if Dir.exists?('empty_directory')
+        FileUtils.rm_rf('empty_directory')
+      end      
```

Comment:
You forget a lot of whitespaces at the end. I removed it by myself, but please, take care with this next time. I recommend you to use git diff before push, you can see this kind of problems :)